netty/src/main/java/io/grpc/transport/netty/ProtocolNegotiators.java

```diff
@@ -86,49 +87,59 @@ public static ProtocolNegotiator tls(final SslContext sslContext,
     Preconditions.checkNotNull(sslContext, "sslContext");
     Preconditions.checkNotNull(inetAddress, "inetAddress");
 
-    final ChannelHandler sslBootstrapHandler = new ChannelHandlerAdapter() {
```

Comment:
Ah right ... not sure I've ever seen it done in real code but I do recall it being possible.